tests/scripts/tg/run_tg_demo_tests.sh

```diff
@@ -102,8 +102,8 @@ run_tg_llama3_70b_dp_tests() {
 
   echo "LOG_METAL: Running run_tg_llama3_70b_dp_tests"
 
-  llama70b=/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/
-  LLAMA_DIR=$llama70b MESH_DEVICE=TG pytest models/tt_transformers/demo/simple_text_demo.py --timeout 1000; fail+=$?
+  llama70b=/mnt/MLPerf/tt_dnn-models/llama/Llama3.1-70B-Instruct/
```

Comment:
This was Llama3.3 before; should not change it to 3.1 in this PR.
```suggestion
  llama70b=/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/
```